windows/NSISCopyRegistryKey/registry.nsh

```diff
@@ -0,0 +1,166 @@
+!ifndef REGISTRY_NSH
```

Comment:
This code has no "watermark" -- it needs some declaration of authorship, licence, etc.

Reply:
It should also be under the same license as nsProcess. Maybe we should just add it here?

Reply:
Done now. I hope this comment is ok.